Lib/test/test_opcache.py

```diff
@@ -1245,6 +1245,14 @@ def f(o, n):
         f(test_obj, 1)
         self.assertEqual(test_obj.b, 0)
 
+# gh-127274: BINARY_SUBSCR_GETITEM will only cache __getitem__ methods that
```

Comment:
This would be removable after https://github.com/python/cpython/pull/128012 is merged.